test/helpers/gasCost.js

```diff
@@ -7,12 +9,13 @@ export const gasCost = gas =>
     1000 * gas / gasLimit
   ) / 10}% of limit)`;
 
-export const txGasCost = tx => gasCost(tx.receipt.gasUsed);
+export const txGasCost = tx => {
+  invariant(
```

Comment:
Why not `expect(tx).to.have.property('receipt')` ?

Reply:
I use `invariant` as a way for checking for developer errors and expect is for test expections. I can unify it, I don't really care :D

Reply:
Me neither. Just curious about it.